apps/server/src/mindmap/dto/create-mindmap.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { IsNotEmpty, IsOptional, IsString } from "class-validator";
+
+export class CreateMindmapDto {
+
+    @IsString()
+    @IsOptional()
+    name: string;
+
+    @IsString()
+    @IsOptional()
+    path: string;
```

Comment:
Just check if we need this at the time of creation. As `path` is formed at the backend, we might not need this property here.